seedfarmer/mgmt/deploy_utils.py

```diff
@@ -113,7 +115,11 @@ def _get_module_info(args: Dict[str, Any]) -> None:
                 )
 
         params = [
-            {"account_id": target_account_region["account_id"], "region": target_account_region["region"]}
+            {
+                "account_id": target_account_region["account_id"],
+                "region": target_account_region["region"],
+                "role_prefix": target_account_region.get("role_prefix", "/"),
```

Comment:
(Leaving this as a somewhat random spot.)

Any particular reason we provide the default (`/`) in many places instead of only at the edges and then require that it be defined?

Reply:
On it